Tests/SkillFunctionalTests/SimpleHostBotToDialogSkillTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using TranscriptTestRunner;
+using TranscriptTestRunner.XUnit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SkillFunctionalTests

```

Comment:
I think that it may be good to start grouping these individual tests into a single folder now that we are starting moving into creating folders for the scenarios. 
We need them for now to develop the testrunner functionality but we will replace them with scenarios tests once that's done.

Reply:
If you agree, we'll move these tests to a _LegacyTests_ folder in the PR for the refactor of the DirectLineClient (ISSUE #198)